bundles/org.openhab.binding.energenie/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="thing-type:energenie:config">
+		<parameter name="host" type="text" required="true">
+			<label>IP-Address</label>
+			<description>The IP address of this device</description>
+			<context>network-address</context>
+		</parameter>
+		<parameter name="password" type="text" required="true">
+			<label>Password</label>
+			<description>The password of this device</description>
+			<default>1</default>
```

Comment:
Thisdefault is not inline with the implementation in the configuration class (`""`). Also: If you provide a default, `required="true"` is not needed.

Reply:
This is just the factory default